docs/oauth2.md

```diff
@@ -0,0 +1,30 @@
+Oauth2
+======
+
+Oauth2 is a django app that will can be used to cache your `django-oauth-toolkit`
+access token model.
+
+
+Usage
+-----
+To start caching your api access tokens, add `django_toolkit.oauth2` to your
```

Comment:
Ain't no need for doing it now. When I read that docs section again I started to think I got it wrong.